elftools/elf/enums.py

```diff
@@ -16,20 +16,23 @@
     ELFCLASS64=2
 )
 
+
```

Comment:
These added newlines don't look necessary...

Reply:
From PEP 8,
> Surround top-level function and class definitions with two blank lines.

So I think of these `Enum` as a 'top-level definition'. I remove these then?

Reply:
Yeah, it's not very consistent throughout the code. In this particular file it was all one line. I suggest you don't modify them in your PR as it's unrelated. I can make it more consistent separately later